orc8r/cloud/go/services/metricsd/obsidian/models/swagger.v1.yml

```diff
@@ -1149,18 +1149,12 @@ definitions:
           type: string
       match:
         type: object
-        properties:
-          label:
-            type: string
-          value:
-            type: string
+        additionalProperties:
+          type: string
       match_re:
         type: object
-        properties:
-          label:
-            type: string
-          value:
-            type: string
```

Comment:
It looks like both of them do have the properties `label` and `value` on the Go side, and this is not touched in this PR. Why did you change the interface?

Reply:
As discussed offline, it seems the generated types in the backend are not used, and the new interface is correct.